### PR TITLE
Fix access priv

### PR DESCRIPTION
**Pull request by bennieswart**

### CODE REVIEW

File: pglast/printers/ddl.py [hunk 1]
```diff
@@ -14,7 +14,14 @@
 
 @node_printer("AccessPriv")
 def access_priv(node, output):
-    output.print_node(node.priv_name)
+    if node.priv_name is Missing:
+        output.write('ALL PRIVILEGES')
+    else:
+        output.write(node.priv_name.value.upper())
```
Comment (lelit) [inline pglast/printers/ddl.py]: This is obviously subjective, but I'm not sure if the list of privileges should be considered SQL keywords and thus in uppercase... I usually opted to keep whatever comes out from the PG parser as is.

Comment (bennieswart) [reply pglast/printers/ddl.py]: Your call, ultimately. If one looks at the manual (https://www.postgresql.org/docs/12/sql-grant.html) those terms are always referred to in upper case.

Keep case or capitalize?

Comment (bennieswart) [reply pglast/printers/ddl.py]: I would agree that the field names (`UPDATE (Field1, Field2)`) should be kept as-is, but the `UPDATE` is pretty much always in upper case in my experience.

Comment (lelit) [reply pglast/printers/ddl.py]: Ok then, keep the upper().

**Pull request merged**